### ISSUE TITLE
Hyperlinks not working from extension's "Watched portals" list

### ISSUE THREAD

**Issue by mattozan** [metzen/ingress-notify#14]
I cannot click on the portal titles in order to jump to them on the Ingress Intel map. They appear correctly formatted, but nothing happens when I click on one:

```
http://www.ingress.com/intel?latE6=38539161&lngE6=-121504585&z=16
From: chrome-extension://bdlnageoicfcjdmkcljkompgphkeeghp/popup.html
```

The "X" _does_ work to remove a portal from the list. Just not the portal location hyperlink. If I inspect the element and click the link from the Developer Tools Window, that works as well:

```
<a href="http://www.ingress.com/intel?latE6=38539161&amp;lngE6=-121504585&amp;z=16">
<span class="title ng-binding">Alligator Sculpture</span>
</a>
```

Not sure why the link doesn't function just from the pop-down window, or how I can further troubleshoot...

![github](https://f.cloud.github.com/assets/583480/61252/282df3a8-5c3c-11e2-9254-4399eaf40ea2.png)

**Comment by ichthyos** [comment]
For now, you can work around this by going to https://ingress-notify.appspot.com.

I'll fix this by adding a target attribute to the links to open them in a new tab.

**Comment by ichthyos** [comment]
This has been fixed in 0c03378bdeb16f74c1919b85506b6446642304fc. The next time metzen updates the extension in the Chrome Web Store, the links should be fixed.

**Comment by metzen** [comment]
Pushed the fix in 1.0.11, on Jan 31st.
